QuadradiusR/scripts/right_side_panel.gd

```diff
@@ -0,0 +1,29 @@
+extends PanelContainer
+
+onready var user1Input = $Control/User1Input
+onready var user2Input = $Control/User2Input
+
+onready var user1DecorationActive = $Control/User1DecorationActive
+onready var user2DecorationActive = $Control/User2DecorationActive
+
+
+func _ready():
+	user1Input.text = Context.username
+	user2Input.text = Context.get_other_player()['username']
+	
+	_update_current_player(Context.game_state.get_current_player())
+
+func _update_current_player(id: String):
+	if id == Context.user_id:
+		user1DecorationActive.visible = true
+		user2DecorationActive.visible = false
+	else:
+		user1DecorationActive.visible = false
+		user2DecorationActive.visible = true
```

Comment:
If I recall correctly, the active decoration is bigger than inactive, so you could just leave inactive one permanently visible and put the active one on top of it.

Reply:
That is how it's actually done

Reply:
my bad